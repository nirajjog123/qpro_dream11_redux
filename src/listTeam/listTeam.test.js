import React from "react";
import "jest-dom/extend-expect";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";
import ConnectedApp, { ListTeam } from "./listTeam";

configure({ adapter: new Adapter() });

function setup() {
  const props = {
  //  handleDelete: jest.fn()
  };

  const enzymeWrapper = shallow(<ListTeam {...props} />);

  return {
   // props,
    enzymeWrapper
  };
}

describe("<DetailTeam/>", () => {
  const { enzymeWrapper, props } = setup();

  it("it should render properly ", () => {
    expect(enzymeWrapper).toMatchSnapshot();
  });
});
