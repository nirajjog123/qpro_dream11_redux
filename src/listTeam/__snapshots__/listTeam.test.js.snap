// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DetailTeam/> it should render properly  1`] = `ShallowWrapper {}`;
